### PR TITLE
JIRA-8 Configure docker for application

### PR DESCRIPTION
**Pull request by peterholman**
Adds docker config for running the app.

### CODE REVIEW

File: client/.env
Comment (peterholman) [inline client/.env:1]: Realised we don't actually need this at the moment as we have no default environment variables.

File: client/environment/.env.dev
Comment (peterholman) [inline client/environment/.env.dev:1]: This file is not needed as dev will just use our docker config.

File: client/postcss.config.js
Comment (peterholman) [inline client/postcss.config.js:1]: Moved this config to `package.json`.

File: .github/workflows/run-client-checks.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       - name: setup node
         uses: actions/setup-node@v4
         with:
-          node-version: '20.13.1'
+          node-version: '20.15.1'
```
Comment (peterholman) [inline .github/workflows/run-client-checks.yml:18]: Bumped node to latest lts.

**Review by MikesCS50 — APPROVED** [review]

**Pull request merged**